app/api/websocket.py

```
@@ -184,6 +188,37 @@ def traced_tts_synthesize(sentence: str, tts_language_code: str):
+        sentence: 합성할 문장
+        tts_language_code: 언어 코드 (예: "ko-KR", "ja-JP")
+        voice_id: Cartesia 음성 ID 오버라이드 (Phase 2: 마스코트별 음성). None이면 기본 voice 사용.
+
+    Returns:
+        (audio_bytes, audio_format) — audio_format은 Unity 클라이언트에 전달되는 포맷 문자열
+    """
+    if cartesia_tts is not None:
+        try:
+            audio = await cartesia_tts.synthesize_async(sentence, tts_language_code, voice_id=voice_id)
+            return audio, "pcm_s16le"
+        except Exception as exc:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)